tests/index.js

```diff
@@ -292,6 +292,36 @@ describe('DigitalLink', () => {
       expect(dl.getKeyQualifier(ai)).to.equal(value);
     });
 
+    it('should set the key identifiers in the right order', () => {
+      const values = {
+        'gtin':{
+          'ai':'01',
+          'value':'12345678'
+        },
+        'lot':{
+          'ai':'10',
+          'value':'211'
+        },
+        'ser':{
+          'ai':'21',
+          'value':'2121'
+        },
+        'cpv':{
+          'ai':'22',
+          'value':'122113'
+        },
+      };
+
+      dl.setDomain('https://gs1.evrythng.com');
+      dl.setIdentifier(values.gtin.ai, values.gtin.value);
+      dl.setKeyQualifier(values.lot.ai, values.lot.value);
+      dl.setKeyQualifier(values.cpv.ai, values.cpv.value);
+      dl.setKeyQualifier(values.ser.ai, values.ser.value);
+
+      expect(dl.isValid()).to.equal(true);
+
```

Comment:
Does this test really check the ordering? Or is that performed by `isValid`?

Reply:
isValid returns false if the order is not correct.